### PR TITLE
Fix AuthorizationPolicy Doc

### PR DESCRIPTION
**Pull request by smwyzi**

### CODE REVIEW

File: security/v1beta1/authorization_policy.proto [hunk 1]
```diff
@@ -311,7 +311,7 @@ message AuthorizationPolicy {
 }
 
 // Rule matches requests from a list of sources that perform a list of operations subject to a
-// list of conditions. A match occurs when at least one source, operation and condition
+// list of conditions. A match occurs when at least one source, operation and all condition
```
Comment (yanniszark) [inline security/v1beta1/authorization_policy.proto:314]: nit: conditions

Comment (smwyzi) [reply security/v1beta1/authorization_policy.proto]: done

**Comment by smwyzi** [comment]
@yangminzhu hi，I saw the release-notes_api Job failed，any tips？ thanks~

**Review by yangminzhu — APPROVED** [review]

**Comment by yangminzhu** [comment]
@istio/technical-oversight-committee can anyone take a look at the PR? just a simple doc fix, thanks.

File: security/v1beta1/authorization_policy.proto [hunk 2]
```diff
@@ -311,7 +311,7 @@ message AuthorizationPolicy {
 }
 
 // Rule matches requests from a list of sources that perform a list of operations subject to a
-// list of conditions. A match occurs when at least one source, operation and condition
+// list of conditions. A match occurs when at least one source, operation and all conditions
```
Comment (smawson) [inline security/v1beta1/authorization_policy.proto:314]:
```suggestion
// list of conditions. A match occurs when at least one source, operation and list of conditions
```

Comment (smawson) [reply security/v1beta1/authorization_policy.proto:314]: Actually that's not right, let me reword again.

Comment (smawson) [reply security/v1beta1/authorization_policy.proto:314]: OK changed my suggestion, that should be good.

Comment (smwyzi) [reply security/v1beta1/authorization_policy.proto]: @smawson done，thanks

**Comment by smwyzi** [comment]
@smawson thanks for your suggestion, fixed.

**Review by nrjpoddar — APPROVED** [review]

**Pull request merged**